.github/workflows/building.yml

```diff
@@ -0,0 +1,47 @@
+name: Building
+
+on:
+  push:
+    branches: [ kawaii ]
+  pull_request:
+    branches: [ kawaii ]
+
+jobs:
+  build-windows:
+    runs-on: windows-2019
+
+    strategy:
+      matrix:
+        node-version: [ 14.x ]
```

Comment:
@acidicMercury8 may you upgrade to 17.x? Does it depend on anything?

Reply:
@PeyTy official Node.js action uses LTS releases. We can use 16.x as a maximum